docs/src/fs.rst

```diff
@@ -218,7 +218,8 @@ API
 
 .. c:function:: int uv_fs_read(uv_loop_t* loop, uv_fs_t* req, uv_file file, const uv_buf_t bufs[], unsigned int nbufs, int64_t offset, uv_fs_cb cb)
 
-    Equivalent to :man:`preadv(2)`.
+    Equivalent to :man:`preadv(2)`. if the `offset` argument is `-1`, then
```

Comment:
```suggestion
    Equivalent to :man:`preadv(2)`. If the `offset` argument is `-1`, then
```

Reply:
Thanks! Should have paid more attention when I copy-pasted...